guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -93,12 +93,14 @@ ifdef::foreman-el,katello,satellite[]
 ----
 endif::[]
 ifdef::satellite[]
+----
+# dnf module enable {dnf-modules} postgresql:12 ruby:2.7
```

Comment:
IMHO you could remove "satellite" in line 87.

Reply:
Addressed :+1: